pyp2rpm/convertor.py

```diff
@@ -281,43 +283,57 @@ def metadata_extractor(self):
 
     @property
     def client(self):
-        """XMLRPC client for PyPI. Always returns the same instance.
+        """JSON client for PyPI. Always returns the same instance.
 
         If the package is provided as a path to compressed source file,
         PyPI will not be used and the client will not be instantiated.
 
         Returns:
-            XMLRPC client for PyPI or None.
+            JSON client for PyPI or None.
```

Comment:
When does this return None?

Reply:
If pyp2rpm is called on a local archive file, the pypi client won't be used.

Reply:
Right. When `self.pypi` is false.

Reply:
Is there an implication that I'm missing?

Reply:
No. It was just a but confusing to me, so I've asked. But it is not wrong.